fed/barriers.py

```diff
@@ -358,7 +365,11 @@ def start_recv_proxy(
         logging_level=logging_level,
         retry_policy=retry_policy,
     )
-    recver_proxy_actor.run_grpc_server.remote()
+    try:
+        recver_proxy_actor.run_grpc_server.remote()
```

Comment:
`remote()` method wouldn't raise any exception.

Reply:
Therefore the except block would never get reached.

Reply:
Indeed. Thx for the hint! Has changed to raise at the remote side and return err msg to driver.